README.md

```diff
@@ -115,6 +115,7 @@ Parameter | Description | Default
 `images.csiClusterDriverRegistrarContainer` | CSI Cluster Driver Registrar Container image | `quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.1`
 `images.csiExternalProvisionerContainer` | CSI External Provisioner Container image | `storageos/csi-provisioner:v1.0.1`
 `images.csiExternalAttacherContainer` | CSI External Attacher Container image | `quay.io/k8scsi/csi-attacher:v1.0.1`
+`ìmages.csiLivenessProbeContainer` | CSI Liveness Probe Container Image | `quay.io/k8scsi/livenessprobe:v1.0.1`
```

Comment:
We need to add this in the CRD as well `deploy/crds/storageos_v1_storageoscluster_crd.yaml` and `deploy/olm/storageos/storageoscluster.crd.yaml`.
I've found some tools like `yt`(yaml processor) to reduce this duplication. Have to automate replicating the changes in all the related files.

Reply:
:) just noticed and pushed

Reply:
@avestuk let's do the TLS options in a separate PR